pkg/langserver/autotype_win.go

```diff
@@ -20,6 +20,12 @@ var (
 		Modifiers: win32.ModCtrl,
 		KeyCode:   'I',
 	}
+	// AutotypeSSCHotkey is the hotkey to trigger auto-typing when in the SSC
+	AutotypeSSCHotkey = &win32.Hotkey{
+		ID:        1,
```

Comment:
Each registered hotkey needs a unique ID. So insteat of re-using 1 here, it should be 4.
(Also, to keep it neatly ordered, that hotkey with id 4 should mmove to the end of the list)